apps/drupal/webpack.config.js

```diff
@@ -49,9 +58,13 @@ const prod = {
 };
 
 module.exports = particle(
+  // app
   { shared, dev, prod },
+  // Default design system
+  APP_DESIGN_SYSTEM,
+  // Use extract css
   {
     cssMode: 'extract',
-    entry: 'app-drupal',
+    entry: APP_NAME, // Called out specifically because 2 entry points
```

Comment:
is this second entry needed due to the css extraction setup? otherwise it's duplicating the entry in shared, yeah?

Reply:
![image](https://user-images.githubusercontent.com/295127/47391776-97e8f080-d6cf-11e8-9810-f00ee55db594.png)

We have 2 entry points in drupal webpack so that jQuery gets bundled into its own file. The "entry" setting seen in the options sent over to the `particle()` function is which entry point `@babel/polyfill` gets prepended to. In this case we're saying "we need babel polyfill automatically prepended to the big ol' custom blob of javascript, but not the jQuery blob"